Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -318,7 +318,8 @@ bool GLContextWidget::load_file(Core::DeprecatedFile& file)
         return false;
     }
 
-    auto new_mesh = m_mesh_loader->load(file);
+    // FIXME: Propagate errors
+    auto new_mesh = m_mesh_loader->load(MUST(String::from_deprecated_string(file.filename())), MUST(Core::File::adopt_fd(file.leak_fd(), Core::File::OpenMode::Read)));
```

Comment:
Please use `release_value_but_fixme_should_propagate_errors()` instead of `MUST()` when something should be propagated.